_posts/2024-11-22-release-2.8.0.md

```diff
@@ -0,0 +1,219 @@
+before upgrading (Python >= 3.10).
+
+
+## Notable changes
+
+For a full list of changes, bugfixes and deprecations see the [CHANGELOG][].
+
+#### Enhancements:
+
+* Added guess_TopologyAttrs() API to the Universe to handle attribute
+  guessing (PR #3753)
+* Added the DefaultGuesser class, which is a general-purpose guesser with
+  the same functionalities as the existing guesser.py methods (PR #3753)
+* Introduce parallelization API to `AnalysisBase` and to `analysis.rms.RMSD` class
+  (Issue #4158, PR #4304)
+* Add `analysis.DSSP` module for protein secondary structure assignment, based on [pydssp](https://github.com/ShintaroMinami/PyDSSP)
+* Improved performance of PDBWriter (Issue #2785, PR #4472)
+* Added parsing of arbitrary columns of the LAMMPS dump parser. (Issue #3504)
+* Implement average structures with iterative algorithm from
+  DOI 10.1021/acs.jpcb.7b11988. (Issue #2039, PR #4524)
+* Add support for TPR files produced by Gromacs 2024.1 (PR #4523)
+
+#### Fixes:
+
+ * Fix Bohrium (Bh) atomic mass in tables.py (PR #3753)
+ * Catch higher dimensional indexing in GroupBase & ComponentBase (Issue #4647)
+ * Do not raise an Error reading H5MD files with datasets like
+   `observables/<particle>/<property>` (part of Issue #4598, PR #4615)
+ * Fix failure in double-serialization of TextIOPicklable file reader.
+   (Issue #3723, PR #3722)
+ * Fix failure to preserve modification of coordinates after serialization,
+   e.g. with transformations
+   (Issue #4633, PR #3722)
+ * Fix PSFParser error when encoutering string-like resids
+   (Issue #2053, Issue #4189 PR #4582)
+ * Convert openmm Quantity to raw value for KE and PE in OpenMMSimulationReader.
+ * Atomname methods can handle empty groups (Issue #2879, PR #4529)
+ * Fix bug in PCA preventing use of `frames=...` syntax (PR #4423)
+ * Fix `analysis/diffusionmap.py` iteration through trajectory to iteration
+   over `self._sliced_trajectory`, hence supporting
+   `DistanceMatrix.run(frames=...)` (PR #4433)
+
+
+#### Changes:
+
+ * Relicense code contributions from GPLv2+ to LGPLv2.1+
+   and the package from GPLv3+ to LGPLv3+ (PR #4794)
+ * only use distopia < 0.3.0 due to API changes (Issue #4739)
+ * The `fetch_mmtf` method has been removed as the REST API service
+   for MMTF files has ceased to exist (Issue #4634)
+ * MDAnalysis now builds against numpy 2.0 rather than the
+   minimum supported numpy version (PR #4620)
+
+
+
+#### Deprecations:
+
+## Author statistics
+
+This release was the work of 22 contributors, **10** of which are **new contributors**.
+
+Our **new contributors** are:
+- @aditya292002
+- @pstaerk
+- @kainszs
+- @SampurnaM
+- @leonwehrhan
+- @PythonFZ
+- @kurtmckee
+- @laksh-krishna-sharma
+- @MattTDavies,
+- @talagayev
+
+## Acknowledgements
+
+MDAnalysis thanks [NumFOCUS][] for its continued support as our fiscal sponsor and 
+the [Chan Zuckerberg Initiative][] for supporting MDAnalysis under EOSS4 and EOSS5 awards.
+
+— The MDAnalysis Team
```

Comment:
This may be up for discussion, but as the guy that has to go through really tedious releases, I was wondering if it would be ok for me to be the person signing off on release things on behalf of the team? 

What I mean here is that it's always a major technical undertaking that doesn't get a mention in the notes/changelog because it happens outside the scope of it - so it's multiple FTE days of hidden work unless we don't make it hidden (happy to go with alternative solutions or be completely shot down)